addon/templates/components/bootstrap-power-select.hbs

```diff
@@ -1,6 +1,7 @@
 {{#bootstrap/control-wrapper errors=errors classNames=classNames ariaLabelledBy=true srOnly=srOnly label=label as |control|}}
+  {{control.label elementId=control.inputId srOnly=srOnly label=label}}
```

Comment:
We definitely need `elementId=control.inputId label=label`, the `srOnly` is passed through.

Reply:
Should we update label so that we don't need to do that?